### PR TITLE
chore: update pnpm-workspace

### PR DESCRIPTION
**Pull request by aladdin-add**
不要将 tests 作为 workspace

##### Checklist

- [ ] `npm test` passes
- [ ] tests are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

- any feature?
- close https://github.com/umijs/qiankun/ISSUE_URL

### CODE REVIEW

File: pnpm-workspace.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 packages:
-  - "packages/**/*"
+  - "packages/*"
```
Comment (kuitos) [inline pnpm-workspace.yaml:2]: 这样会漏掉 ui-bindings 目录下的子包。
可以改成：
```
  - "packages/**/*"
  - "!./packages/**/tests/**"
```

**Review by kuitos — CHANGES_REQUESTED** [review]

File: pnpm-workspace.yaml (hunk 1 above)
Comment (aladdin-add) [reply pnpm-workspace.yaml:2]: 或者改成：
```yaml
packages:
  - "packages/*"
  - "packages/ui-bindings/*"
```
?

Comment (kuitos) [reply pnpm-workspace.yaml:2]: 也可以

Comment (kuitos) [inline pnpm-workspace.yaml:2]:
```suggestion
  - "packages/*"
  - "packages/ui-bindings/*"
```

**Comment by kuitos** [comment]
lock file 是不是得重新生成一下？ @aladdin-add

**Comment by aladdin-add** [comment]
更新了～

**Pull request merged**